tests/plenary/popup_spec.lua

```diff
@@ -160,3 +303,4 @@ describe("plenary.popup", function()
     end)
   end)
 end)
+-- vim:sw=2 ts=2 et
```

Comment:
This should be removed cause it can mess up other peoples environments, better to keep it in your config.

Somewhere like `path/to/your/config/after/ftplugin/lua.lua` if you only want to set it for lua files.

I leave you an example of using this in my config [for Golang](https://github.com/AlejandroSuero/.dotfiles/blob/main/roles/neovim/files/after/ftplugin/go.lua)

Reply:
So the use of a modeline is not the style in plenary.nvim. It's nice to have stuff local instead of global. Oh well.  I'll take it out

While I'm figuring out lua... I'll use
```vim
autocmd BufRead */plenary.nvim/*.lua set sw=2 ts=2 sts=2 noet
```

Reply:
> This should be removed cause it can mess up other peoples environments, better to keep it in your config.

BTW, there's `set nomodeline` or `set modelines=0` to avoid modelines modifying your environment.

Reply:
I just came across `:help editorconfig`
> Think of it like 'modeline' for an entire (recursive) directory.

Adding such a file to the plenary.nvim repo might be useful.